includes/class-wc-siftscience-format-order.php

```diff
@@ -123,47 +123,47 @@ private function convert_order_status( WC_Order $order ) {
 		}
 
 		/**
-		 * @param $order
-		 * @param string $type
-		 * '$address'  => array(
-		 *		'$name'         => 'Bill Jones',
-		 *		'$phone'        => '1-415-555-6041',
-		 *		'$address_1'    => '2100 Main Street',
-		 *		'$address_2'    => 'Apt 3B',
-		 *		'$city'         => 'New London',
-		 *		'$region'       => 'New Hampshire',
-		 *		'$country'      => 'US',
-		 *		'$zipcode'      => '03257'
-		 *	),
+		 *
+		 * @param  WC_Order $o
+		 * creates shipping address 
+		 *
 		 * @return array
-		 */
-
-		private function create_address( WC_Order $order, $type = 'shipping' ) {
-			if( $type == 'billing' ){
-				$address_object = array(
-					'$name'      => $order->get_billing_first_name() . ' ' . $order->get_billing_last_name(),
-					'$phone'     => $order->get_billing_phone(),
-					'$address_1' => $order->get_billing_address_1(),
-					'$address_2' => $order->get_billing_address_2(),
-					'$city'      => $order->get_billing_city(),
-					'$region'    => $order->get_billing_state(),
-					'$country'   => $order->get_billing_country(),
-					'$zipcode'   => $order->get_billing_postcode()
-				);
-			} elseif($type == 'shipping'){
-				$address_object = array(
-					'$name'      => $order->get_shipping_first_name() . ' ' . $order->get_shipping_last_name(),
-					'$company'   => $order->get_shipping_company(),
-					'$address_1' => $order->get_shipping_address_1(),
-					'$address_2' => $order->get_shipping_address_2(),
-					'$city'      => $order->get_shipping_city(),
-					'$region'    => $order->get_shipping_state(),
-					'$country'   => $order->get_shipping_country(),
-					'$zipcode'   => $order->get_shipping_postcode()
-				);
-			}			
-			$address_object = apply_filters( 'wc_siftscience_create_address', $address_object, $order, $type );
-			return $address_object;
+		 *
+		*/
+		private function create_shipping_address(WC_Order $order){
+			$shipping_address = array(
+								'$name'      => $order->get_formatted_shipping_full_name(),
```

Comment:
The indentation is way off for this. It should be one tab more than the previous line.